docs/stoploss.md

```diff
@@ -27,55 +21,99 @@ So this parameter will tell the bot how often it should update the stoploss orde
 
 For example, simplified math:
 
 * the bot buys an asset at a price of 100$
-* the stop loss is defined at 2%
-* the stop loss would get triggered once the asset dropps below 98$
+* the stop loss is defined at -10%
+* the stop loss would get triggered once the asset dropps below 90$
 * assuming the asset now increases to 102$
-* the stop loss will now be 2% of 102$ or 99.96$
-* now the asset drops in value to 101$, the stop loss will still be 99.96$ and would trigger at 99.96$.
+* the stop loss will now be -10% of 102$ = 91,8$
+* now the asset drops in value to 101$, the stop loss will still be 91,8$ and would trigger at 91,8$.
 
 In summary: The stoploss will be adjusted to be always be 2% of the highest observed price.
 
-### Custom positive stoploss
+### Trailing stop loss, custom positive loss
 
-It is also possible to have a default stop loss, when you are in the red with your buy, but once your profit surpasses a certain percentage, the system will utilize a new stop loss, which can have a different value.
-For example your default stop loss is 5%, but once you have 1.1% profit, it will be changed to be only a 1% stop loss, which trails the green candles until it goes below them.
+It is also possible to have a default stop loss, when you are in the red with your buy (buy - fee), but once you hit possitive result the system will utilize a new stop loss, which can have a different value.
+For example your default stop loss is -10%, but once you have more than 0% profit (example 0.1%) a different trailing stoploss will be used.
 
-Both values require `trailing_stop` to be set to true.
+Both values require `trailing_stop` to be set to true and trailing_stop_positive with a value.
 
 ``` python
-    trailing_stop_positive = 0.01
-    trailing_stop_positive_offset = 0.011
+    stoploss = -0.10
+    trailing_stop = True
+    trailing_stop_positive = 0.02
 ```
 
-The 0.01 would translate to a 1% stop loss, once you hit 1.1% profit.
-Before this, `stoploss` is used for the trailing stoploss.
+For example, simplified math:
 
-Read the [next section](#trailing-only-once-offset-is-reached) to keep stoploss at 5% of the entry point.
+* the bot buys an asset at a price of 100$
+* the stop loss is defined at -10%
+* the stop loss would get triggered once the asset dropps below 90$
+* assuming the asset now increases to 102$
+* the stop loss will now be -2% of 102$ = 99,96$
+* now the asset drops in value to 101$, the stop loss will still be 99,96$ and would trigger at 99,96$
 
-!!! Tip
-    Make sure to have this value (`trailing_stop_positive_offset`) lower than minimal ROI, otherwise minimal ROI will apply first and sell the trade.
+The 0.02 would translate to a -2% stop loss.
+Before this, `stoploss` is used for the trailing stoploss.
 
-### Trailing only once offset is reached
+### Trailing stop loss only once the trade has reached a certain offset.
```

Comment:
```suggestion
### Trailing stop loss only once the trade has reached a certain offset
```

I wouldn't use a sentence as title ... (so without trailing .).

Reply:
Hmm it's copy paste from the stop loss type list above.. i will check on that